server/controllers/appointments/deleteAppointments.js

```diff
@@ -0,0 +1,18 @@
+const { deleteAppointmentsQueries } = require('../../database/queries');
+
+const deleteAppointments = async (req, res, next) => {
+  try {
+    const { appointmentId } = req.params;
+
+    const { rows } = await deleteAppointmentsQueries(appointmentId);
+    return res.json({
+      statusCode: 200,
+      message: 'appointment deleted successfully',
+      data: rows,
+    });
```

Comment:
I think the status code of this response after elimaiting data from it should be 204 .
```
Status Codes

200 OK - Some people think a delete function of any kind should return the deleted element, so a representation of the deleted element can be included in the response body.
204 No Content - The most fitting status code for this case. It’s better to reduce traffic and simply tell the client the deletion is complete and return no response body (as the resource has been deleted).
202 Accepted - If the deletion is asynchronous and takes some time, which is the case in distributed systems, it can be appropriate to return this code with some information or URL to tell the client when it will be deleted.
```
Please view this [link](https://www.moesif.com/blog/technical/api-design/Which-HTTP-Status-Code-To-Use-For-Every-CRUD-App/) to get more .